src/test/java/com/github/tomakehurst/wiremock/DeadlockTest.java

```diff
@@ -125,8 +123,6 @@ private void downloadContentAndMeasure(String urlDir, String expectedBody) throw
   }
 
   private String httpGetContent(HttpURLConnection connection) throws IOException {
-    try (InputStream is = connection.getInputStream()) {
-      return IOUtils.toString(is, StandardCharsets.UTF_8);
-    }
+    return getStringFromInputStream(connection.getInputStream());
```

Comment:
It's only test code, but any reason for removing the try with resources here?

Reply:
done